index.js

```diff
@@ -2,22 +2,21 @@ let util = require("util");
 let fs = require("fs");
 let readFile = util.promisify(fs.readFile);
 
-exports.middleware = (manifestPath, { caching } = {}) => {
-	if(caching === undefined) {
-		caching = (process.env.NODE_ENV === "production");
-	}
+let PRODUCTION = process.env.NODE_ENV === "production";
```

Comment:
Note that this is only evaluated on startup now, but middleware initialization effectively also happens at startup I suppose? (I don't think we want to do this anew with each request.)